### DISCUSSION TITLE
Node v14.20.0 resolves npm v8.19.1 by default

### DISCUSSION THREAD

**Discussion by thelovekesh** [nodejs/help#4978] [Q&A]
While installing Node v14.20.0 it installs NPM v8.19.1 by default while with Node v14.19.3 it installs NPM v6.14.17.

![image](https://user-images.githubusercontent.com/54371619/190684761-f0a25b51-b55a-419d-8057-31948d632bb1.png)

Upon checking releases it doesn't seem there is any commit related to the NPM update. Can someone please explain it's working?

**Comment by aduh95** [comment]
Maybe a bug in nvm? In this case it should be reported to them.

**Reply by thelovekesh** [reply]
But I think NPM is bundled with Node and NVM simply installs Node.

**Reply by aduh95** [reply]
Did you try with the installer from https://nodejs.org? We have no control over what nvm does, it's a separate project, they may have decided to bump the npm version for some reason, I don't know.

FWIW npm v6.14.1 is the on e vendered-in on `v14.x` branch: https://github.com/nodejs/node/blob/v14.x/deps/npm/package.json
